.cz-config.js

```diff
@@ -0,0 +1,59 @@
+const types = [
+  ` feat:     A new feature`,
+  ` fix:      A bug fix`,
+  ` docs:     Documentation only changes`,
+  ` style:    Changes that do not affect the meaning of the code
+             (white-space, formatting, missing semi-colons, etc)`,
+  ` refactor: A code change that neither fixes a bug nor adds a feature`,
+  ` perf:     A code change that improves performance`,
+  ` test:     Adding missing tests`,
+  ` revert:   Revert to a commit`,
+  ` WIP:      Work in progress`,
+  ` chore:    Changes to the build process or auxiliary tools
+             and libraries such as documentation generation`
+];
+
+const messages = {
+  type: `Select the type of change that you're committing:`,
+
+  subject: `Write a SHORT, IMPERATIVE tense description of the change:\n`,
+
+  body: `Provide a LONGER description of the change (optional). Use "|" to break new line:\n`,
+
+  breaking: `Describe any BREAKING CHANGES (optional):\n`,
+
+  footer: ` List any RELATED ISSUES to this change (optional).\n`,
+
+  confirmCommit: `Are you sure you want to proceed with the commit above?`
+};
+
+const scopes = [
+  "card",
+  "clientpicker",
+  "createaddress",
+  "hermit",
+  "slices",
```

Comment:
what are slices?

Reply:
I think they're the little table panel things that @bucko13 put in.

Reply:
a "slice" is the name we're using for a cross section of braid at a particular path